eventing/sources/generator/SourcesTemplate.gomd

```diff
@@ -0,0 +1,39 @@
+{{- /* This will be interpreted given a yamlSources object. */ -}}
+
+# Knative Event Sources
+
+This is a non-exhaustive list of Event sources for Knative. Inclusion in this list **does not**
+imply an endorsement, nor does it imply any level of support.
+
+
+## Sources
+
+These are sources that are installed as `CRD`s.
+
+Name | Status | Support | Description
+--- | --- | --- | ---
+{{ range .Sources -}}
+[{{ .Name }}]({{ .Url }}) | {{ .Status }} | {{ .Support }} | {{ .Description }}
+{{- end }}
+
+
+## Meta Sources
```

Comment:
Not sure whether we need to generate these, since we shouldn't end up with a new meta source for each new source. 😬

Reply:
> since we shouldn't end up with a new meta source for each new source.

I think I agree with you, there should be far fewer meta sources than sources. MetaSources are more useful for source authors, not end users that just want to find a working source.

As for listing these, I think it is useful to have a place where we list all these meta sources. It doesn't have to be in the same place as all the end user usable sources, but for now I don't think it is problematic to have it here, especially as `Containers` refer to these by name.